runtimes/java/select-java-version.md

```diff
@@ -7,7 +7,7 @@ tags:
 
 Simply set the environment variable **CC_JAVA_VERSION** to the version you want
 
-Accepted values are `8`, `11`, `13` or `graalvm-ce`
+Accepted values are `7` to `14` or `graalvm-ce` (with GraalVM being based on OpenJDK 11.0).
```

Comment:
I think you can also add the graalvm version here which is `20.1.0`

Reply:
Makes sense, yes. I'll change it.